### ISSUE TITLE
Why is there a 🍐(pear emoji) in the menu bar?

### ISSUE THREAD

**Issue by chadlavi** [withfig/fig#1736]
### Sanity checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [x] I have run `fig restart` and tested again (tell us if that fixed it)

### Description:

There's a 🍐(pear emoji) in the menu bar icon for Fig. It's cut off, and I don't understand why it would be there in the first place.

Here's a screenshot of it:
<img width="82" alt="image" src="https://user-images.githubusercontent.com/7226642/187304423-62ad0659-5066-4f9d-8a99-8d5f954a1418.png">

What is going on here?

### Environment

```yaml
# Fig Diagnostics
## Fig details:
  - Fig version: Version 1.0.60 (B507) [U.S.]
  - Bundle path: /Applications/Fig.app
  - Autocomplete: true
  - Settings.json: true
  - Accessibility: true
  - Number of specs: 198
  - Symlinked dotfiles: false
  - Only insert on tab: false
  - Keybindings path: 
  - Installation Script: true
  - PseudoTerminal Path: /opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/Users/chad/.fig/bin:/Users/chad/.local/bin
  - SecureKeyboardInput: false
  - SecureKeyboardProcess: <none>
## Hardware Info:
  - Model Name: MacBook Air
  - Model Identifier: MacBookAir10,1
  - Chip: Apple M1
  - Cores: 8 (4 performance and 4 efficiency)
  - Memory: 16 GB
## OS Info:
  - macOS 13.0.0 (22A5331f)
## Environment:
  - User Shell: /bin/zsh
  - Current Directory: /Users/chad
  - CLI Installed: true
  - Executable Location: /Users/chad/.fig/bin/fig
  - Current Window ID: 653/% (com.apple.Terminal)
  - Active Process: zsh (6269) - /dev/ttys002
  - Terminal: terminal
  - Environment Variables:
    - FIG_INTEGRATION_VERSION=8
    - FIG_TERM=1
    - FIG_TERM_VERSION=5.1.0
    - PATH=/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/Users/chad/.fig/bin:/Users/chad/.local/bin
    - TERM=xterm-256color
    - TERM_SESSION_ID=2ADC2662-B9CE-4A94-9166-2CFDF0E18D5D
    - FIG_PID=6269
## Integrations:
  - SSH: false
  - TMUX: false
  - iTerm: application is not present.
  - Hyper: application is not present.
  - Visual Studio Code: installed!
  - Docker: false
```

**Comment by grant0417** [comment]
Any ideas @mschrage

**Comment by mschrage** [comment]
This is a placeholder asset from ages ago. I've removed it. Fix should go live in v1.0.61.
